repro.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Why is this needed? Afaik it gives us no benefits?

Reply:
One of the goals of this project is to be distribution independent. NixOS for example doesn't have a bash binary in `/bin`.

Reply:
Nix is sufficiently weird that I'm not really going to care about them unless someone is intentionally trying to run `repro` on NixOS.

Reply:
Afaik, NixOS patches shebangs and asks their contributors specifically not to make PR's to every piece of software ever existed to solve their own invented problem. Python does the same imo `/usr/bin/python` not `/usr/bin/env python`.

Someone is welcome to package repro for NixOS and then patch it there.

Also plain using `env` causes problems as https://discourse.nixos.org/t/how-do-you-run-a-bash-script/10141/8